src/lib/eina/eina_debug_timer.c

```diff
@@ -95,17 +96,16 @@ _monitor(void *_data EINA_UNUSED)
    ret = epoll_ctl(epfd, EPOLL_CTL_ADD, event.data.fd, &event);
    if (ret) perror("epoll_ctl/add");
 #ifdef EINA_HAVE_PTHREAD_SETNAME
-# ifndef __linux__
-   pthread_set_name_np
-# else
-   pthread_setname_np
-# endif
-     (pthread_self(), "Edbg-tim");
+
+ eina_thread_name_set(eina_thread_self(), "Edbg-tim");
```

Comment:
Maybe remove empty line and a little indentation?
```suggestion
    eina_thread_name_set(eina_thread_self(), "Edbg-tim");
```

Reply:
How many spaces do you use? here is 3